kernel/dev/rtc/rtc.cpp

```diff
@@ -101,7 +106,8 @@ void read_rtc() {
         last_century = rtc_century;
 
         // Make sure an update isn't in progress
-        while (rtc_get_update_in_progress());
+        while (rtc_get_update_in_progress())
```

Comment:
See previous comment.

Reply:
I think I should format it part by part rather than all code.